simulation_ws/src/hello_world_simulation/launch/empty_world.launch.py

```diff
@@ -21,24 +21,31 @@
 from ament_index_python.packages import get_package_share_directory
 import launch
 
+
 def generate_launch_description():
     hello_world = get_package_share_directory('hello_world_simulation')
     gazebo_ros = get_package_share_directory('gazebo_ros')
 
     gazebo_client = launch.actions.IncludeLaunchDescription(
-	launch.launch_description_sources.PythonLaunchDescriptionSource(
-            os.path.join(gazebo_ros, 'launch', 'gzclient.launch.py')),
+        launch.launch_description_sources.PythonLaunchDescriptionSource(
+            os.path.join(gazebo_ros, 'launch', 'gzclient.launch.py')
+        ),
         condition=launch.conditions.IfCondition(launch.substitutions.LaunchConfiguration('gui'))
      )
     gazebo_server = launch.actions.IncludeLaunchDescription(
         launch.launch_description_sources.PythonLaunchDescriptionSource(
             os.path.join(gazebo_ros, 'launch', 'gzserver.launch.py'))
     )
 
-    turtlebot3_description_reduced_mesh = get_package_share_directory('turtlebot3_description_reduced_mesh')
+    turtlebot3_description_reduced_mesh = get_package_share_directory(
+        'turtlebot3_description_reduced_mesh')
     turtlebot3_description_reduced_mesh_launch = launch.actions.IncludeLaunchDescription(
         launch.launch_description_sources.PythonLaunchDescriptionSource(
-            os.path.join(turtlebot3_description_reduced_mesh, 'launch', 'spawn_turtlebot.launch.py'))
+            os.path.join(
+                turtlebot3_description_reduced_mesh,
+                'launch',
+                'spawn_turtlebot.launch.py'
+                ))
```

Comment:
I'm kind of surprised at this kind of formatting. I was expecting these brackets have one less indent or be on the previous line. 🤷‍♂️